### ISSUE TITLE
2025-03-19-15-2

### ISSUE THREAD

**Issue by lxr-astro** [lxr-astro/ChatArxiv#10]
ther high maximum SFE nor high stochasticity alone can explain the high abundances of luminous galaxies seen by JWST. Extending the analysis of the UVLF and 2PCF to $z>9$ measured from wider surveys will be crucial in breaking degeneracies.

**Summary**: 

- (1):这篇文章的研究背景是解释由詹姆斯韦伯太空望远镜（JWST）揭示的早期宇宙中亮丽和巨大星系丰富的现象。

- (2):过去的方法主要集中在调整一维统计数据以匹配模型，存在问题是这种方法存在内在模糊和单一性。提出的方法通过结合两点相关函数的星系聚集和紫外发光函数的测量数据，以及提出SFE-Mh关系，解决了单点统计测量带来的模糊性问题。

- (3):本文的贡献在于发展了一个理论框架，通过星系聚类和半径建模结合来提供早期宇宙星系-暗物质晕的观测约束。

- (4):本文提出的研究方法包括测量两点相关函数（2PCF）和紫外发光函数（UVLF），并通过对H$\alpha$和[OIII]发射体的样本进行拟合来推断SFE和暗物质晕的关系。

- (5):该方法的目标是解释JWST发现的明亮星系的高丰度，通过模型验证得出了暗物质晕质量与星系聚类的关系，并提出了SFE随暗物质晕质量的变化规律，支持了研究的目标。

# galaxies
## Velocity Structure Correlations between the Nebular, Molecular, and Atmospheric Gases in the Cores of Four Cool Core Clusters
- **Url**: http://arxiv.org/abs/2503.14486v1
- **Authors**: ['Muzi Li', 'B. R. McNamara', 'Alison L. Coil', 'Marie-Joelle Gingras', 'Fabrizio Brighenti', 'H. R. Russell', 'Prathamesh D. Tamhaneh', 'S. Peng Oh', 'Serena Perrotta']
- **Abstrat**: We investigate the velocity structure of nebular gas in the central galaxies of four clusters: Abell 1835, PKS 0745-191, Abell 262, and RXJ0820.9+0752, using data from the Keck Cosmic Web Imager (KCWI). Velocity structure functions (VSFs) of the [OII] emission line are compared to VSFs of molecular clouds observed with the Atacama Large Millimeter/submillimeter Array (ALMA). Apart from Abell 262 where the gas is located in a circumnuclear disk, the nebular gas in the remaining galaxies lies in off-nuclear filamentary structures with VSFs steeper than the Kolmogorov slope. This steepening may be plausibly attributed to gravity although other factors, such as magnetic stresses and bulk motion,} may be significant. The VSFs of CO and [OII] emission are similar in RXJ0820 and Abell 262, indicating close coupling of the nebular and molecular gases. In contrast, the nebular and molecular gases are differentiated on most scales in PKS 0745 and Abell 1835. This discrepancy is likely due to the radio-AGN churning the gas. We compare the scale-dependent velocity amplitudes of the hot atmospheres constrained by X-ray surface brightness fluctuation analysis using Chandra observations to the nebular VSFs. The large-scale consistency in Abell 1835 and RXJ0820 is consistent with condensation from the hot atmospheres. {We explore substantial systematic biases, including projection effects, windowing, and smoothing effects when comparing VSFs using different telescopes and instruments.

**Summary**: 

- (1):文章研究背景为**星系团**研究中心气体的速度结构相关性。

- (2):过去的方法包括使用X射线表面亮度波动分析约束热大气层的速度振幅，存在的问题是不清楚热大气层与星系气体的速度结构联系，提出的方法不同之处在于比较星系气体的速度结构函数(VSF)与分子云的VSF，并探讨它们之间的联系，提出方法有明显的动机。

- (3):本文的贡献在于探索四个**冷核星系团**中心星系的星系气体速度结构，并将其与分子云和大气气体的速度结构进行对比。

- (4):本文提出的研究方法包括使用Keck Cosmic Web Imager (KCWI)观测星系内部的星系气体的速度结构，并与Atacama Large Millimeter/submillimeter Array (ALMA)观测的分子云VSF进行比较。

- (5):本文方法在研究星系气体速度结构与分子云、大气气体速度的相关性方面取得了显著成果，展示了星系团核心冷核星系的动力学特征。

## Constraints on the early Universe star formation efficiency from galaxy clustering and halo modeling of H$α$ and [O III] emitters
- **Url**: http://arxiv.org/abs/2503.14280v1
- **Authors**: ['Marko Shuntov', 'Pascal A. Oesch', 'Sune Toft', 'Romain A. Meyer', 'Alba Covelo-Paz', 'Louise Paquereau', 'Rychard Bouwens', 'Gabriel Brammer', 'Viola Gelli', 'Emma Giovinazzo', 'Thomas Herard-Demanche', 'Garth D. Illingworth', 'Charlotte Mason', 'Rohan P. Naidu', 'Andrea Weibel', 'Mengyuan Xiao']
- **Abstrat**: We develop a theoretical framework to provide observational constraints on the early Universe galaxy-halo connection by combining measurements of the UV luminosity function (UVLF) and galaxy clustering via the 2-point correlation function (2PCF). We implemented this framework in the FRESCO and CONGRESS JWST NIRCam/grism surveys by measuring the 2PCF of spectroscopically selected samples of H$\alpha$ and [OIII] emitters at $3.8<z<9$ in 124 arcmin$^2$ in GOODS-N and GOODS-S. By fitting the 2PCF and UVLF at $3.8<z<9$ we inferred that the H$\alpha$ and [OIII] samples at $\langle z \rangle \sim4.3, 5.4$ and $7.3$ reside in halos of masses of log$(M_{\rm h}/$M$_{\odot}) = 11.5$, $11.2$, $11.0$ respectively, while their galaxy bias increases with redshift with values of $b_{\rm g} = 4.0$, $5.0$, $7.6$. These halos do not represent extreme overdense environments at these epochs. We constrain the instantaneous star formation efficiency (SFE), defined as the ratio of the star formation rate over the baryonic accretion rate as a function of halo mass. The SFE rises with halo mass, peaks at $\sim20\%$ at $M_{\rm h} \sim 3 \times 10^{11}\, M_{\odot}$, and declines at higher halo masses. The SFE-$M_{\rm h}$ shows only a mild evolution with redshift with tentative indications that low mass halos decrease but the high mass halos increase in efficiency with redshift. The scatter in the $M_{\rm UV}-M_{\rm h}$ relation, quantified by $\sigma_{\rm UV}$, implies stochasticity in the UV luminosities of $\sim 0.7$ mag, relatively constant with z. Extrapolating our model to $z>9$ shows that a constant SFE-$M_{\rm h}$ fixed at $z=8$ cannot reproduce the observed UVLF and neither high maximum SFE nor high stochasticity alone can explain the high abundances of luminous galaxies seen by JWST. Extending the analysis of the UVLF and 2PCF to $z>9$ measured from wider surveys will be crucial in breaking degeneracies.

**Summary**: 

- (1):这篇文章的研究背景是解释James Webb Space Telescope (JWST)在早期宇宙中发现的亮且巨大星系丰富的问题;

- (2):过去的方法主要集中在模拟 UV 光度函数 (UVLF) 和星系聚类的模型，但存在解释模型与观测结果不一致的问题。本文提出的方法不同于现有方法，通过测量两点相关函数 (2PCF) 和 UVLF，结合星系和暗物质晕的关联性，以观察约束早期宇宙星系形成效率。该方法的动机合理，并针对现有问题进行了解决;

- (3):本文的贡献在于提供了理论框架，通过实际数据约束早期宇宙星系与暗物质晕之间的连接；

- (4):本文提出的研究方法涉及测量Hα和[OIII]发射体的2PCF，并通过拟合2PCF和UVLF，推断不同红移下的晕体质量、星系偏置和瞬时星形成效率的关系；

- (5):本文的方法在探索星系形成效率和晕体特征之间的关系方面取得了进展，但在支持其目标方面还需进一步验证。

## Are CMB derived cosmological parameters affected by foregrounds associated to nearby galaxies?
- **Url**: http://arxiv.org/abs/2410.24026v2
- **Authors**: ['Facundo Toscano', 'Frode K. Hansen', 'Diego Garcia Lambas', 'Heliana Luparello', 'Pablo Fosalba', 'Enrique Gaztañaga']
- **Abstrat**: We perform cosmological parameters estimation on Planck Cosmic Microwave Background (CMB) maps masking the recently discovered foreground related to nearby spiral galaxies. In addition, we also analyse the association between these foreground regions and recent claims of cosmological causal horizons in localized CMB parameter estimates. Our analysis shows consistent cosmological parameter values regardless of the masking approach, though reduced sky areas introduce larger uncertainties. By modelling the new extragalactic foreground, we identify a resemblance with local parameter variation maps with a statistical significance at the 3 sigma level, suggesting that a simplified foreground model partially accounts, (40-50)% correlation with 15% uncertainty, for the observed causal horizons. These findings add new evidence to the existence of the new foreground associated with large spiral galaxies and show that estimates of cosmological parameters on smaller patches on the sky can be largely affected by these foregrounds, but that the parameters taken over the full sky are unaltered.

**Summary**: 

- (1):本文的研究背景是关于Planck宇宙微波背景（CMB）图像中近邻螺旋星系引起的前景效应。

- (2):过去的方法存在前景污染和天文学偏差，提出的方法通过模拟新的星系外前景，提供了更好的天文参数估计。该方法合理。
  
- (3):本文的贡献是证实大型螺旋星系引起的新前景可以影响CMB宇宙学参数的估计。
  
- (4):本文采用在Planck CMB图像中遮蔽近邻螺旋星系前景的方法进行宇宙学参数估计，同时建立新的星系外前景模型。
  
- (5):本文的方法实现了一致的宇宙学参数估计，并通过模型指出新前景与本地参数变化图的相似性，提出简化前景模型可部分解释观测到的因果地平线。性能支持了他们的目标。

# machine learning
## An Effective Theory of Bias Amplification
- **Url**: http://arxiv.org/abs/2410.17263v4
- **Authors**: ['Arjun Subramonian', 'Samuel J. Bell', 'Levent Sagun', 'Elvis Dohmatob']
- **Abstrat**: Machine learning models can capture and amplify biases present in data, leading to disparate test performance across social groups. To better understand, evaluate, and mitigate these biases, a deeper theoretical understanding of how model design choices and data distribution properties contribute to bias is needed. In this work, we contribute a precise analytical theory in the context of ridge regression, both with and without random projections, where the former models feedforward neural networks in a simplified regime. Our theory offers a unified and rigorous explanation of machine learning bias, providing insights into phenomena such as bias amplification and minority-group bias in various feature and parameter regimes. For example, we observe that there may be an optimal regularization penalty or training time to avoid bias amplification, and there can be differences in test error between groups that are not alleviated with increased parameterization. Importantly, our theoretical predictions align with empirical observations reported in the literature on machine learning bias. We extensively empirically validate our theory on synthetic and semi-synthetic datasets.

**Summary**: 

- (1):这篇文章研究的背景是机器学习模型可能捕捉并放大数据中存在的偏见，从而导致社会群体之间出现不同的测试表现。

- (2):过去的方法显示出偏见放大的问题，而提出的方法在于通过精确的理论框架解释模型设计选择和数据分布特性如何相互作用来放大偏见，并且提供了对偏见放大和少数群体偏见的解释。这一方法是有动机的，因为其理论框架可以更好地理解和干预偏见问题。

- (3):这篇论文的贡献在于提供了一种有效的理论，能够解释机器学习中偏见放大的现象，并且在合成和半合成数据集上对理论进行了广泛的实证验证。

- (4):在这篇论文中提出的研究方法是在岭回归的框架下提供一个精确的分析理论，以及探讨了前馈神经网络的简化情形，通过调整正则化惩罚或训练时间来避免偏见放大。

- (5):该方法在合成和半合成数据集上进行了广泛的实证验证，获得了与机器学习偏见领域文献中观察到的经验现象相符的实验结果。

## La Méthode du Gradient Proximé
- **Url**: http://arxiv.org/abs/2503.14479v1
- **Authors**: ['Patrick L. Combettes']
- **Abstrat**: English version of abstract for "The Proximal Gradient Method": The proximal gradient method is a splitting algorithm for the minimization of the sum of two convex functions, one of which is smooth. It has applications in areas such as mechanics, inverse problems, machine learning, image reconstruction, variational inequalities, statistics, operations research, and optimal transportation. Its formalism encompasses a wide variety of numerical methods in optimization su
